src/lib/datasource/topic.entity.ts

```diff
@@ -0,0 +1,36 @@
+import { EntitySchema } from 'typeorm';
+import type { TopicMessage } from './topic-message.entity';
+
+export interface Topic {
+  id: string;
+  projectId: string;
+  file: string;
+  messages: TopicMessage[];
+}
+
+export const TopicEntity = new EntitySchema<Topic>({
+  name: 'topic',
+  columns: {
+    id: {
+      type: 'varchar',
+      primary: true,
+      length: 255
+    },
+    projectId: {
+      type: 'varchar',
+      length: 255,
+      nullable: false
+    },
+    file: {
+      type: 'text',
+      nullable: false
+    }
+  },
+  relations: {
+    messages: {
+      target: 'topic-message',
+      type: 'one-to-many'
```

Comment:
Existe uma vírgula faltando ao final da linha 32 (`type: 'one-to-many'`). A falta dessa vírgula pode causar um erro de sintaxe no TypeScript.
Garantir a correta sintaxe do código é essencial para evitar erros de compilação e execução.